src/wallet/wallet_rpc_server.cpp

```diff
@@ -1730,10 +1664,9 @@ namespace tools
       m_wallet->rescan_spent();
       return true;
     }
-    catch (const std::exception &e)
+    catch (const std::exception& e)
```

Comment:
You have several spurious whitespace changes like this. Please try to avoid them (fine for this time since it's nearby code changes).